examples/worlds/gimbal.sdf

```diff
@@ -0,0 +1,390 @@
+<?xml version="1.0" ?>
+<!--
+  Ignition gazebo demo using IMU sensor for Gimbal
```

Comment:
Can you add some lines explaining the user what commands they can try, and what output they should expect to see? See how we do it in other examples, for example:

https://github.com/ignitionrobotics/ign-gazebo/blob/17f63c497e7fb81dca5f87708ba8b2128db67b31/examples/worlds/ackermann_steering.sdf#L3-L11